recipes/uncommitted/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "uncommitted" %}
+{% set version = "1.6" %}
+{% set md5 = "87a357627ed16a9f02498cea9c61cbb0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  md5: {{ md5 }}
+
+build:
+  number: 0
+  script: python setup.py install
+  skip: True  # [win and py3k]
```

Comment:
This doesn't have dependencies outside of Python, right? So that issue isn't relevant. Please see PR ( https://github.com/conda-forge/uncommitted-feedstock/pull/1 ) to enable all Windows builds.